node/src/test/scala/com/wavesplatform/state/diffs/ci/InvokeScriptTransactionDiffTest.scala

```diff
@@ -2355,6 +2357,214 @@ class InvokeScriptTransactionDiffTest
       compileContractFromExpr(expr, V5)
     }
 
+    def contract1(otherAcc: Address, alias: Alias): DApp = {
```

Comment:
This duplicates 
https://github.com/wavesplatform/Waves/pull/3336/files#diff-7a87dd14a82b2cd0e58fddefc26230665e583d933ab5ba4da3ea943f13f4a7f1R2463